src/include/recurringClosures.js

```diff
@@ -232,7 +232,7 @@ WMEAC.refreshClosureListFromSelection = function ()
         {
             var blackList=[];
             W.model.roadClosures.getObjectArray(function (c) {
-                return c.segID==W.selectionManager.getSelectedFeatures()[0].model.attributes.id;
+                return c.segID==W.selectionManager.getSelectedFeatures()[0].attributes.wazeFeature.id;
```

Comment:
I would change this to W.selectionManager.getSelectedDataModelObjects()[0].id

Reply:
Looks simpler. Thanks.

Reply:
This will be the same problem. The id at the root of that returned data is null.

Reply:
changed to use getSelectedDataModelObjects()[]  .atttributes.id here